modules/dynamic-provisioning-change-default-class.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * storage/dynamic-provisioning.adoc
+
+[id="change-default-storage-class-{context}"]
+= Changing the defaulte StorageClass
+
+If you are using GCE and AWS, use the following process to change the 
+default StorageClass. This process assumes you have two StorageClass
```

Comment:
This sentence drops off.

Reply:
Finished the thought that this sentence begins.